cmd/notation-azure-kv/key.go

```diff
@@ -1,43 +1,18 @@
 package main
 
 import (
+	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
-	"io"
 	"os"
 
 	"github.com/Azure/notation-azure-kv/internal/signature"
 	"github.com/notaryproject/notation-go/plugin/proto"
-
-	"github.com/urfave/cli/v2"
 )
 
-var describeKeyCommand = &cli.Command{
-	Name:   string(proto.CommandDescribeKey),
-	Usage:  "Azure key description",
-	Action: runDescribeKey,
-	Flags: []cli.Flag{
-		&cli.StringFlag{
-			Name:      "file",
-			Usage:     "request json file",
-			TakesFile: true,
-			Hidden:    true,
-		},
-	},
-}
-
-func runDescribeKey(ctx *cli.Context) error {
-	var r io.Reader
-	if f := ctx.String("file"); f != "" {
-		var err error
-		r, err = os.Open(f)
-		if err != nil {
-			return err
-		}
-	} else {
-		r = os.Stdin
-	}
+func runDescribeKey(ctx context.Context) error {
+	r := os.Stdin
 	var req proto.DescribeKeyRequest
 	err := json.NewDecoder(r).Decode(&req)
```

Comment:
We can just replace `r` with `os.Stdin`.

Reply:
Updated.